docs/plugins/ingest-attachment.asciidoc

```diff
@@ -52,7 +52,7 @@ The node must be stopped before removing the plugin.
 | `field`                | yes       | -                | The field to get the base64 encoded field from
 | `target_field`         | no        | attachment       | The field that will hold the attachment information
 | `indexed_chars`        | no        | 100000           | The number of chars being used for extraction to prevent huge fields. Use `-1` for no limit.
-| `properties`           | no        | all              | Properties to select to be stored. Can be `content`, `title`, `name`, `author`, `keywords`, `date`, `content_type`, `content_length`, `language`
+| `properties`           | no        | all properties   | Properties to select to be stored. Can be `content`, `title`, `name`, `author`, `keywords`, `date`, `content_type`, `content_length`, `language`
```

Comment:
I agree. I wondered first if we should add somehow a new column with the expected format, or add it within the comment ro as an example which I did at the end.
I cand replace `Properties to select` with `Array of properties to select`?